markdoc/tags/Next.markdoc.ts

```diff
@@ -0,0 +1 @@
+export { comment, head, link, script } from '@markdoc/next.js/tags';
```

Comment:
Can we just export them directly from `markdoc/tags/index.ts`

Reply:
That would work too, but I was just going on the file naming convention from the code block here:

https://markdoc.dev/docs/nextjs#built-in-next.js-tags